.travis.yml

```diff
@@ -6,7 +6,7 @@ script:
   - xcrun instruments -t 'Blank' -l 10 -w "iPhone 6s (10.0)"
   - swift build
   - swift test
-  - xcodebuild -scheme JSONUtilities -configuration Debug -sdk iphonesimulator10.0 -destination "OS=10.0,name=iPhone 6s" test -enableCodeCoverage YES
+  - xcodebuild -scheme "JSONUtilities iOS" -configuration Debug -sdk iphonesimulator10.0 -destination "OS=10.0,name=iPhone 6s" test -enableCodeCoverage YES
```

Comment:
Totally - I'm actually going to switch this to one scheme based on [the ideas from this article](http://promisekit.org/news/2016/08/Multiplatform-Single-Scheme-Xcode-Projects/) (& prompted by another project where I made similar scheme / project changes).  I'll certainly add tvOS as a test target & if your tests don't have UIKit, I'll add macOS too.  watchOS is another matter as it doesn't have XCTest, but I'll be happy to add a build check on here too.